src/tests/commonTests.ts

```diff
@@ -196,6 +196,11 @@ interface FocusableOptions {
    * selector used to assert the focused shadow DOM element
    */
   shadowFocusTargetSelector?: string;
+
+  /**
+   * Used to delay setFocus call in ms.
+   */
+  delay?: number;
```

Comment:
If this is for the open/close transition, I'd suggest using the `skipAnimation` helper instead of introducing a delay argument. If it's something specific to `focus-trap` behavior, can you add more info about this to the documentation?

Reply:
@jcfranco the problem is the `delayInitialFocus` is set to true.

I tried `delayInitialFocus: !Build.isTesting,` but the problem is that its not returning true.

```
{"testing":false,"browser":true,"dev":false,"server":false} 
    Location: http://localhost:3333/build/p-371d7086.entry.js:15:2306
```